engine/execution/ingestion/engine.go

```diff
@@ -1050,15 +1031,7 @@ func (e *Engine) handleComputationResult(
 }
 
 // save the execution result of a block
-func (e *Engine) saveExecutionResults(
-	ctx context.Context,
-	executableBlock *entity.ExecutableBlock,
-	stateInteractions []*delta.Snapshot,
-	events []flow.Event,
-	serviceEvents []flow.Event,
-	txResults []flow.TransactionResult,
-	startState flow.StateCommitment,
-) (*flow.ExecutionResult, error) {
+func (e *Engine) saveExecutionResults(ctx context.Context, executableBlock *entity.ExecutableBlock, spockSnapshots []*delta.SpockSnapshot, events []flow.Event, serviceEvents []flow.Event, txResults []flow.TransactionResult, startState flow.StateCommitment) (*flow.ExecutionReceipt, error) {
```

Comment:
did we do this style change on purpose?

Reply:
Some automatic refactoring I guess, thanks for noticing! Reversed